charts/portworx/templates/portworx-service.yaml

```diff
@@ -8,8 +8,20 @@ metadata:
 spec:
   selector:
     name: portworx
+  type: NodePort
```

Comment:
Oh.. was this broken before, since we missed NodePort?

Reply:
@zoxpx no if we dont specify NodePort, it will be defaulted to ClusterIP, and currently with all communication to the portworx-service is internal within the k8s cluster it should work. shouldnt be broken.